static/css/styles.css

```diff
@@ -79,28 +52,81 @@ header h1 a {
   --code-bg: rgb(55, 56, 62);
   --hljs-bg: rgb(46, 46, 51);
 }
-/* Dark Mode Init - No Animation */
-.dark-mode-init .toggle-slider {
-  transition: none !important;
+
+.dark-mode-off:root {
+  --bg-color: #f0efea;
+  --text-color: #141413;
+  --border-color: #e6e4dd;
+  --footnote-bg-color: rgba(0, 0, 0, 0.08);
+  --footnote-hover-bg-color: rgba(0, 0, 0, 0.15);
+  --tooltip-bg-color: #f9f9f9;
+  --tooltip-text-color: #333;
+  --tooltip-shadow-color: rgba(0, 0, 0, 0.15);
+  --subscribe-form-bg: transparent;
+  --subscribe-form-text: #141413;
+  --subscribe-button-bg: #141413;
+  --subscribe-button-text: #f0efea;
+  --invert-percentage: 0%;
+  --code-bg: rgb(223, 223, 223);
+  --hljs-bg: rgb(28, 29, 33);
 }
 
-.dark-mode-init .toggle-input:checked + .toggle-label .toggle-slider {
-  transform: translateX(20px);
+@media (prefers-color-scheme: dark) {
+  /* Same as .dark-mode-on:root */
```

Comment:
The media query blocks for dark and light are copies of `.dark-mode-on:root` and `.dark-mode-off:root` respectively.